src/components/RecoveryPhraseInput.tsx

```diff
@@ -68,7 +68,7 @@ export default function RecoveryPhraseInput({
   return (
     <Card
       rounded={true}
-      shadow={showInput ? Shadow.SoftLight : null}
+      shadow={showInput ? null : Shadow.SoftLight}
```

Comment:
is this intentional? looking at the figma, it seems like the shadow is present when the recovery phrase is fully entered but "Restore" is still pressable, which isn't really a processing state.

Reply:
updated in [0ec330a](https://github.com/valora-inc/wallet/pull/5628/commits/0ec330a29c9ba037741e99d50e72a18f0c27f81a) to change to showing the shadow and different colored background when a valid phrase is entered